states/new_jersey/elections.toml

```diff
@@ -3,7 +3,140 @@ date = 2020-07-07
 name = "Congressional & Presidential Primary"
 original_date = 2020-07-07
 
+[20200707.registration]
+received_by = 2020-06-16
+sources = ["https://www.state.nj.us/state/elections/voter-registration.shtml",
+           "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-02-primary-election-052220.pdf"]
+
+[20200707.absentee.application]
+received_by = 2020-06-30
+in_person_by = 2020-07-07T20:00:00
+sources = [
+    "https://www.state.nj.us/state/elections/primary-election-070720.shtml",
+    "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-02-primary-election-052220.pdf"
+]
+
+[20200707.absentee.military_overseas.application]
+online_by = "2020-07-03"
+sources = ["https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-02-primary-election-052220.pdf"]
+
+[20200707.absentee]
+postmarked_by = 2020-07-07
+received_by = 2020-07-14T20:00:00
+in_person_by = 2020-07-07T20:00:00
+sources = ["https://www.state.nj.us/state/elections/assets/pdf/election-results/2020/2020-drop-box-locations.pdf",
+           "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-02-primary-election-052220.pdf"]
+
+[20200707.poll]
+in_person_start = 2020-07-07T06:00:00
+in_person_by = 2020-07-07T20:00:00
+sources = ["https://elections.delaware.gov/calendar/2020ElectionCalendar.pdf",
+           "https://lis.njleg.state.nj.us/nxt/gateway.dll/statutes%2F1%2F19539%2F19749"]
+
+[20200929]
+date = 2020-09-29
+name = "September Special School Board"
+original_date = 2020-09-29
+
+[20200929.registration]
+received_by = 2020-09-08
+in_person_by = 2020-09-08
+sources = ["https://www.state.nj.us/state/elections/voter-registration.shtml",
+           "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-08-september-school-board.pdf"]
+
+[20200929.absentee.application]
+received_by = 2020-09-29
```

Comment:
I think received_by and possibly in_person_by should be 2020-09-22

From the sources: 
September 22 Deadline to apply for a Mail-In Ballot by Mail for September Special School Board Elections(not less than 7 days prior to election)N.J.S.A. 19:63-3